### PR TITLE
Log "Stream Paused" Bar Styling Fix

### PR DESCRIPTION
**Pull request by samblackk**
Attached to https://github.com/astronomer/astronomer/issues/289

Updates the "stream paused" bar to be absolutely positioned under the log container (removes jumpiness) and fades the box in/out depending on if the logs are paused (reduces awkwardness). 

Please r/m

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Env vars
 .env
 
- Logs
-/logs
```
Comment (samblackk) [inline .gitignore]: This directory was hidden via`.gitignore` - I removed it to push these changes.

**Comment by samblackk** [comment]
Updated to remove "pause", allow infinite @ of log items, and fix scroll jank. Scrolling is reversed in the log box - this is a side effect of using a CSS solution to continuously display the bottom of the log items container (because we load in from the bottom). The solution is cleaner than hijacking the scroll via JS.

**Comment by samblackk** [comment]
I can't fix the "flash" that happens when filtering the logs - this is a rooted issue in Orbit with how the HOC components. The flash is a rerender of the entire component because the entire component is wrapped in the HOC. Pinning this as a pain point to fix while reworking how Orbit handles GraphQL.

**Comment by schnie** [comment]
Hmm, I really like having it scroll normally. Went back and forth on that initially and I think its too jarring to have an inversed scroll. It is kinda ugly code-wise though, which bugged me but just couldn't get over the scroll thing so just moved forward with it. Also looks like the scrollbar is on the left side for me :thinking:

**Comment by schnie** [comment]
@samblackk yea we can skip fixing that HOC issue for now.

**Comment by schnie** [comment]
@samblackk any chance we could revert the (inverse) scroll behavior back to how it was, and keep the other fix?

**Comment by samblackk** [comment]
@schnie Scroll is fixed up :)

**Pull request merged**